examples/packages/cache/bbolt/main.go

```diff
@@ -16,15 +16,16 @@ func main() {
 	const myKey, myValue = "myKeyName", "myValue"
 
 	// create a new memory cache
-	c, err := bbolt.New("/Users/jranson/test.bbolt", "trickster")
+	c := bbolt.New("bbolt", "/Users/jranson/test.bbolt", "trickster", nil)
```

Comment:
backlog thought: should these examples go to the package instead of an examples package?

e.g. https://go.dev/blog/examples

Reply:
The idea here was that is easy to click through all of the examples in one place in case one might prove interesting.  I think they are fine to relocate, but we would need a new Markdown doc that curates all of the important examples into a single list / place. It's fine to file an issue for the curated doc and we can address later. But the path should probably just be `"./test.bbolt"` without the absolute home directory. Clearly a copy/paste oversight. Thanks!